R/drive_reveal.R

```diff
@@ -0,0 +1,40 @@
+#' Add columns with pertinent information to your dribble
+#'
+#' @template file-plural
+#' @param what Character.
+#'  * publish
+#'  * permissions
+#'  * trash
+#'  * path
+#'  * mime_type
+#'
+#' @template dribble-return
+#' @export
+#' @examples
+#' \dontrun{
+#' ## Upload a file
+#' file <- drive_upload(R.home('doc/html/logo.jpg'))
+#'
+#' ## Reveal `trash` status
+#' drive_reveal(file, what = "trash")
+#'
+#' ## Reveal `permissions`
+#' drive_reveal(file, what = "permissions")
+#'
+#' ## Reveal `publish` status
+#' drive_reveal(file, what = "publish")
+#'
+#' ## Clean up
+#' drive_rm(file)
+#' }
+drive_reveal <- function(file, what = "path") {
+
+  reveal <- list("publish" = drive_show_publish,
+                 "permissions" = drive_show_permissions,
+                 "trash" = drive_show_trash,
+                 "path" = drive_show_path,
+                 "mime_type" = drive_show_mime_type)[[what]]
```

Comment:
What if `what` has length greater than one? Or doesn't match any of those? `file` also needs some validation.